ui/src/types/pdfjs-viewer.ts

```diff
@@ -1,4 +1,25 @@
-import { PDFDocumentProxy, PDFPageProxy, PDFPageViewport } from "pdfjs-dist";
+import { PDFDocumentProxy, PDFPageProxy } from "pdfjs-dist/types/display/api";
+
+interface PDFPageViewportOptions {
```

Comment:
It is puzzling that @types/pdfjs-dist changed in how it exports the types! Good to see a fix here :-)

Reply:
I'm not really sure what caused this. `pdfjs-dist` also includes typescript definitions itself, and on this stack, the @types packaged seemed like it was being ignored completely. So this import path is where these actual defs from `pdfjs-dist` are located. There's some [confusing communication from the pdfjs devs](https://github.com/mozilla/pdf.js/issues/12384#issuecomment-695071180) about these types only being exported at all for test reasons (yay, JS) but they've kinda become things that all the PDF libraries using PDF.JS expect to work with.

Reply:
> but they've kinda become things that all the PDF libraries using PDF.JS expect to work with.

Fair enough. I do not have too much concern about these types not necessarily being recommended by the PDF.js team as we update the pdf.js version so rarely (i.e., never to date) that I am not concerned about running into breaking changes related to updated types